docs/getting_started.md

```diff
@@ -0,0 +1,89 @@
+# Getting Started
+
+## What is the goal of Inago?
+
+Inago is a deployment tool that manages groups of unit files to deploy them to
+a fleet cluster similar to `fleetctl`. Since `fleetctl` is quite limited, Inago
```

Comment:
I'm not certain I like saying that `fleetctl` is quite limited, without qualifying it. Maybe something explaining that `fleetctl` is not great at handling large numbers of unit files, or sets of unit files?